docs/protocol_method_client.md

```diff
@@ -0,0 +1,54 @@
+A client with protocol methods, often known informally as a "Low Level Client", is generated from autorest.csharp the same as other clients are. 
+
+However a handful of configuration changes are needed to get started.
+
+1. Create a standard "high level" project [as documented](csharp_generate).
+
+2. Add the following key to the readme.md or autorest.md configuration file:
+
+```yaml
+low-level-client: true
+```
+
+3. Define the authentication in configuration 
```

Comment:
This is shared with non LLC. Should we extract? Or should this be in autorest repo?

Reply:
I lean towards autorest repo. Should I file an issue?

Reply:
Yep

Reply:
Done in https://github.com/Azure/autorest/issues/4126

Reply:
Turns out that these are actually autorest.csharp constructs:

```
                autoRest.GetValue<string[]?>("credential-types").GetAwaiter().GetResult() ?? Array.Empty<string>(),
                autoRest.GetValue<string[]?>("credential-scopes").GetAwaiter().GetResult() ?? Array.Empty<string>(),
                autoRest.GetValue<string?>("credential-header-name").GetAwaiter().GetResult() ?? "api-key",
```

Reply:
Do you want me to pull these up to a higher level document that isn't LLC specific?

Reply:
I would leave it in LLC until we switch to the standardized model.